src/DynamoCoreWpf/Views/Preview/Watch3DView.xaml.cs

```diff
@@ -61,6 +60,8 @@ public Viewport3DX View
             get { return watch_view; }
         }
 
+        public HelixWatch3DViewModel ViewModel { get; private set; }
```

Comment:
Ditto here. Changed from `internal` to `public` as I need a handle to this externally as well.